src/XrdAccSciTokens.cc

```diff
@@ -23,6 +23,7 @@
 
 // The status-quo to retrieve the default object is to copy/paste the
 // linker definition and invoke directly.
+XrdVERSIONINFO(XrdAccAuthorizeObject, XrdAccSciTokens);
 extern XrdAccAuthorize *XrdAccDefaultAuthorizeObject(XrdSysLogger   *lp,
```

Comment:
Could you get rid of this extern as it's not used anywhere (it's a holdover) and we eventually want to get rid of it.